PowerEditor/src/NppIO.cpp

```diff
@@ -1194,6 +1194,11 @@ bool Notepad_plus::fileSave(BufferID id)
 				}
 			}
 
+			// Expand any environment variables
+			TCHAR fn_bak_expanded[MAX_PATH] = { '\0' };
+			::ExpandEnvironmentStrings(fn_bak.c_str(), fn_bak_expanded, MAX_PATH);

```

Comment:
I wasn't too worried about checking the return value since most other places in the code tend to ignore it as well. Plus if something does fail it most likely will cause `CopyFile()` to fail.